packages/expo-modules-core/ios/Fabric/EXHostWrapper.h

```diff
@@ -0,0 +1,19 @@
+// Copyright 2024-present 650 Industries. All rights reserved.
+
+#import <UIKit/UIKit.h>
```

Comment:
Breaks compatibility with macOS.
```suggestion
#import <ExpoModulesCore/Platform.h>
```